src/FTPObject.jl

```diff
@@ -1,35 +1,20 @@
-
-function process_response(resp)
-    if isa(resp, RemoteRef)
-        resp = fetch(resp)
-        if(isa(resp, RemoteException))
-            throw(resp)
-        end
-    end
-
-    return resp
-end
-
-
 type FTP
     ctxt::ConnContext
 
-
-
-    function FTP(;host="", block=true, implt=false, ssl=false, ver_peer=true, act_mode=false, user="", pswd="", binary_mode=true)
-        options = RequestOptions(blocking=block, implicit=implt, ssl=ssl,
+    function FTP(;host="", implt=false, ssl=false, ver_peer=true, act_mode=false, user="", pswd="", binary_mode=true)
```

Comment:
I know you didn't change the names of these keywords but some of these keywords could be renamed to be nicer:

`implt` -> `implicit`
`ver_peer` -> `verify`
`act_mode` -> `active`